Tools/update_copyright.sh

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
+
```

Comment:
Don't forget to add the new header here :D

Reply:
Good point, done.